articles/communication-services/quickstarts/email/send-email-smtp/smtp-authentication.md

```diff
@@ -28,7 +28,7 @@ Application developers who build apps that send email using the SMTP protocol ne
 
 ### Creating a custom email role for the Entra application
 
-The Entra application must be assigned a role with both the **Microsoft.Communication/CommunicationServices/Read** and the **Microsoft.Communication/EmailServices/write** permissions on the Azure Communication Service Resource. This can be done either by using the **Contributor** role, or by creating a **custom role**. Follow these steps to create a custom role by cloning an existing role.
+The Entra application must be assigned a role with both the **Microsoft.Communication/CommunicationServices/Read**, **Microsoft.Communication/CommunicationServices/Write**, and the **Microsoft.Communication/EmailServices/write** permissions on the Azure Communication Service Resource. This can be done either by using the **Contributor** role, or by creating a **custom role**. Follow these steps to create a custom role by cloning an existing role.
```

Comment:
A super nit, but is the "w" in "Microsoft.Communication/EmailServices/write" intentionally lower case? The other two permissions have each word capitalized.

Reply:
That's intentional. The permission name uses lower case.